design/FY2019/Interzone Air Wall NFP and Design.md

```diff
@@ -0,0 +1,192 @@
+Air Thermal Boundary - aka Interzone Air Wall
+================
+
+**Michael J. Witte, GARD Analytics, Inc.**
+
+ - Original May 8, 2019
+ - Revision Date
+ 
+
+## Table of Contents ##
+
+[Justification for New Feature](#justification-for-new-feature)
+
+[E-mail and  Conference Call Conclusions](#e-mail-and-conference-call-conclusions)
+
+[Approach](#approach)
+
+[Testing/Validation/Data Sources](#testingvalidationdata-sources)
+
+[Input Output Reference Documentation](#input-output-reference-documentation)
+
+[Engineering Reference](#engineering-reference)
+
+[Example File and Transition Changes](#example-file-and-transition-changes)
+
+[References](#references)
+
+[Design](#design)
+
+## Justification for New Feature ##
+
+Open boundaries between thermal zones occur frequently in buildings. Examples include:
+ - Open plan offices with core and perimeter zones.
+ - Open doorway from one space to another, e.g. hallway opening into a larger space.
+ - Service opening, e.g. serving window between kitchen and dining area.
+ - High spaces where the occupied zone is conditioned differently from the rest, e.g. atrium or auditorium
+
+Open thermal boundaries allow the following heat transfer to occur:
+ - Solar gains and daylighting to pass through as if there is no boundary
+ - Radiant exhange between building surfaces as if there is no boundary.
+ - Distribute radiant internal  gains as if there is no boundary.
+ - Allow airflow and/or mixing to occur between the adjacent zones
+
+This type of boundary occurs frequently in energy models, and is often called an "air wall." 
+EnergyPlus has no direct way to model such a boundary. Common workarounds include some 
+combination of the following:
+ - Near-perfect interzone window
+ - Surface using `Material:InfraredTransparent`
+ - Surface using a very thin highly conductive material
+ - `ZoneMixing` and `ZoneCrossMixing`
+ - AirflowNetwork openings
+
+All of these options have serious limitations, especially the lack of solar/daylight penetration and 
+no direct radiant exchange across the boundary. 
+
+## E-mail and  Conference Call Conclusions ##
+
+n/a
+
+## Approach ##
+
+The proposed approach is to remove the air wall surface entirely from most of the heat transfer calculations
+and create combined superzones for solar, daylighting, and radiant exchange calculations. Air exchange is
+more complicated.
```

Comment:
@mjwitte Not sure why air exchange should be more complicated, since there's an easy solution - just disallow exchange and lump everything together. That should be fine as long as the zone conditions aren't super different, and that's something we can warn about if it happens. For AirflowNetwork it could easily be optional to treat the zones separately or lumped together (provided that there's something to attach a node to).